requirements.txt

```diff
@@ -17,3 +17,8 @@ coverage
 awscli
 flake8
 python-dotenv>=0.5.1
+
```

Comment:
`psycopg2` should be here too

Reply:
Interesting, `pipreqs` missed that. I assume this happened because `psycopg2` is never imported?

Reply:
Yeah I think so. This is why I'm not a fan of `pipreqs`